docs/source/models.rst

```diff
@@ -55,13 +55,13 @@ And its ``MLmodel`` file describes two flavors:
       python_function:
         loader_module: mlflow.sklearn
 
-This model can then be used with any tool that supports *either* the ``sklearn`` or
-``python_function`` model flavor. For example, the ``mlflow models serve`` command can serve a
-model with the ``sklearn`` flavor:
+This model can then be used with any tool that supports either the ``sklearn`` or
+``python_function`` model flavor. However, to serve a model flavor, the ``mlflow models serve`` command
+supports only the ``python_function`` flavor:
 
 .. code-block:: bash
 
-    mlflow models serve my_model
+    mlflow models serve -m my_model
```

Comment:
👍

Reply:
What blurb should we add? It seems like the only supported backend flavors are defined here:
https://github.com/mlflow/mlflow/blob/97c9b9d641ffc9b866c836c9e526277920f28c2a/mlflow/models/flavor_backend_registry.py#L14. That mapping of flavors to flavor backends only supports two. 

To that extent, we mention that you can either serve a model with the `python_function` or the `crate (R Function) flavor`